zebra-state/src/response.rs

```diff
@@ -69,11 +69,27 @@ pub enum Response {
     /// specified block hash.
     BlockHash(Option<block::Hash>),
 
+    /// Response to [`Request::Contains`].
+    BlockLocation(Option<BlockLocation>),
```

Comment:
Can we rename this response and enum? It is very similar to "block locator", which is part of the network protocol and state service.

We could use `KnownBlock` instead?

Reply:
Yeah, renamed in [`7bf56da`](https://github.com/ZcashFoundation/zebra/pull/6335/commits/7bf56da36c09487803f890fb0a9a399085ac3b8f).